.git-blame-ignore-revs

```diff
@@ -1,2 +1,5 @@
 # Apply clang-format to the project.
 16d8647946a4f2b99f0d6b8cc05a83833f85749e
+
+# Apply prettier to the project
+3ef1cecbb1411c3ce4f2e79bca8053e9bcb603c2
```

Comment:
We need to make sure this PR isn't squash merged for this to work, which might need a branch rules change. Alternatively this could be a fast-follow with the commit hash of the squashed commit?

Reply:
It looks like GH will allow a merge commit if needed. In any case, I'll do what it takes to make sure the right commit hash is in place when this gets merged.